tests/unit/integrations/admin/background-indexing-integration-test.php

```diff
@@ -127,9 +168,7 @@ protected function set_up() {
 	public function test_get_conditionals() {
 		static::assertEquals(
```

Comment:
I am confused why this is suddenly a static?

Reply:
This PR just adds an additional test following the pattern of the existing ones. I'm not sure if it's rightfully a static, but maybe it's out of the scope to investigate this here?